pushapkscript/apk.py

```diff
@@ -0,0 +1,72 @@
+from zipfile import ZipFile
+
+from pushapkscript.exceptions import TaskVerificationError
+from pushapkscript.utils import filter_out_identical_values
+
+
+_DIRECTORY_WITH_ARCHITECTURE_METADATA = 'lib/'     # For instance: lib/x86/ or lib/armeabi-v7a/
+_ARCHITECTURE_SUBDIRECTORY_INDEX = len(_DIRECTORY_WITH_ARCHITECTURE_METADATA.split('/')) - 1    # Removes last trailing slash
+
+_EXPECTED_MOZAPKPUBLISHER_ARCHITECTURES = ('armv7_v15', 'x86')
```

Comment:
We might want this configurable per branch at some point, should we ever change which platforms are on m-c and if that rides the trains.  If that happens, we might want to send expected arch down from the decision task, or pushapk can check a file in-tree.  This works for now.